.github/ISSUE_TEMPLATE/devops_ticket.md

```diff
@@ -0,0 +1,24 @@
+---
+name: :boom: DevOps ticket
+about: Help us manage our deployed App.
+labels: devops
+title: :boom: [DevOps]
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-05T12:42:27Z" title="Wednesday, February 5th 2020, 1:42:27 pm +01:00">Feb 5, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,24 @@
+---
+name: :boom: DevOps ticket
+about: Help us manage our deployed App.
+labels: devops
+title: :boom: [DevOps]
```


hey @alina-beck ... I wasn't really able to copy the image here, do you know how I can do that?

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-10T18:26:17Z" title="Monday, February 10th 2020, 7:26:17 pm +01:00">Feb 10, 2020</time>_

---


Not sure what you mean... I usually go to this list: https://gist.github.com/rxaviers/7360908, then select and copy the icon (`💥`) instead of the text (`:boom:`) – it's just a character!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-12T14:43:47Z" title="Wednesday, February 12th 2020, 3:43:47 pm +01:00">Feb 12, 2020</time>_

---


I was able to do this with the :wrench: that you added as a suggestion to my `PR`, but for some reason when I try to copy from the list you shared :point_up_2: it just copies `boom` not the icon, and without the `:`s ... most cases the `:`boom`:` works, but not for creating issue templates.. :worried: